extensions/widgets/treeview/treeview.lcb

```diff
@@ -1865,6 +1890,12 @@ private handler setArrayData(in pData as Array) returns nothing
 	redraw all
 end handler
 
+private handler setFoldState(in pData as Array) returns nothing
+	put pData into mFoldState
```

Comment:
@bwmilby it would probably be a good idea before setting `mFoldState` here to iterate it and check the values are either boolean or a string. If they are a string then parse as boolean and update the value. If not then throw an error.

Reply:
Roger, will update.